bin/env_variables_all.txt

```diff
@@ -58,18 +69,22 @@ DEFINITION_STORE_HOST=http://localhost:4451
 DEFINITION_STORE_IDAM_KEY=AAAAAAAAAAAAAAAA
 DEFINITION_STORE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw,ccd_admin,aac_manage_case_assignment
 DM_STORE_BASE_URL=http://dm-store:8080
+DOCUMENT_STORE_URL=http://dm-store:8080
```

Comment:
Do we need this defined twice or can we just adjust the case-doc code that uses the 'new' value?

Reply:
Will raise a PR to fix in case doc code

Reply:
https://github.com/hmcts/ccd-case-document-am-api/pull/386